Purchases/Misc/SystemInfo.swift

```diff
@@ -47,9 +47,11 @@ class SystemInfo {
     var finishTransactions: Bool
     let platformFlavor: String
     let platformFlavorVersion: String?
+    private let bundle: Bundle
```

Comment:
Style: we try to keep the lets/vars organized by:
public
internal
fileprivate
private

and then the static/non-statics grouped together.